setup.py

```diff
@@ -5,7 +5,7 @@
 
 install_requires = [
     'django-bakery~=0.12.7',
-    'wagtail>=2.10',
+    'wagtail>=2.14',
```

Comment:
I bumped the minimum required version of Wagtail too, but this may be unnecessarily restrictive because we haven't made any changes (yet) that will break comparability with older Wagtail versions.